### PR TITLE
Fix: Scoreboard not showing player count in dungeons

### PR DESCRIPTION
**Pull request by ThatGravyBoat**
## What
Fixes that in dungeons the players title is replaced with a party title so the player count isnt calculated.

## Changelog Fixes
+ Fixed scoreboard player count not displaying in dungeons. - ThatGravyBoat

### CODE REVIEW

**Comment by lunaynx** [comment]
In theory, couldn't this be wrong if some of the party members leave the dungeon during the run?

**Comment by ThatGravyBoat** [comment]
> In theory, couldn't this be wrong if some of the party members leave the dungeon during the run?

it updates accordingly, when I tested someone disconnected so that should mean its the same for someone just doing /hub or something

**Review by CalMWolfs — APPROVED** [review]
worked

File: src/main/java/at/hannibal2/skyhanni/data/HypixelData.kt [hunk 1]
```diff
@@ -67,6 +68,10 @@ class HypixelData {
             "playeramount.guesting",
             "^\\s*(?:§.)*Guests (?:§.)*\\((?<amount>\\d+)\\)\\s*$"
         )
+        private val dungeonPartyAmountPattern by patternGroup.pattern(
```
Comment (hannibal002) [inline src/main/java/at/hannibal2/skyhanni/data/HypixelData.kt:74]: please add regex test

**Review by hannibal002 — CHANGES_REQUESTED** [review]
worked fine for me

**Review by hannibal002 — APPROVED** [review]

**Pull request merged**